hal/src/nRF52840/sleep_hal.cpp

```diff
@@ -379,6 +387,10 @@ static void fpu_sleep_prepare(void) {
     SPARK_ASSERT((fpscr & 0x07) == 0);
 }
 
+#ifdef DEBUG_BUILD
```

Comment:
I assume this is here only for going through this function more easily with a debugger. Let's remove it.